File: framework/security/src/test/java/top/bettercode/simpleframework/security/impl/EmpAuth.java
package top.bettercode.simpleframework.security.impl;

import java.lang.annotation.Documented;
import java.lang.annotation.ElementType;
import java.lang.annotation.Inherited;
import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;
import java.lang.annotation.Target;
import top.bettercode.simpleframework.security.ConfigAuthority;


@Target({ElementType.METHOD, ElementType.TYPE})
@Retention(RetentionPolicy.RUNTIME)
@Inherited
@Documented
@ConfigAuthority("emp")
public @interface EmpAuth {

}

File: security/src/test/kotlin/top/bettercode/summer/security/EmpAuth.kt
package top.bettercode.summer.security

import top.bettercode.summer.security.authorize.ConfigAuthority
import java.lang.annotation.Inherited

@Target(AnnotationTarget.FUNCTION, AnnotationTarget.PROPERTY_GETTER, AnnotationTarget.PROPERTY_SETTER, AnnotationTarget.CLASS)
@Retention(AnnotationRetention.RUNTIME)
@Inherited
@MustBeDocumented
@ConfigAuthority("emp")
annotation class EmpAuth  